datanode/gateway/graphql/resolvers.go

```diff
@@ -481,6 +481,18 @@ func (r *myDepositResolver) CreditedTimestamp(ctx context.Context, obj *types.De
 
 type myQueryResolver VegaResolverRoot
 
+func (r *myQueryResolver) PositionsConnection(ctx context.Context, filter *v2.PositionsFilter, pagination *v2.Pagination) (*v2.PositionConnection, error) {
```

Comment:
We've got a bit of a mix of these but maybe just call this `Positions` ? The 'connection' part is a bit redundant and as I remember we only did it to disambiguate from the non-paged versions which don't exist (would be nice to go through and rename all the XXXConnection to XXX at some point too.

Reply:
From a usability/docs perspective, I second this!

Reply:
That's a good point, we can do it for this one, but the other XXXConnections will need us to go through the same deprecation/rename process as it's going to be quite a major breaking change if we don't deprecate.

Reply:
yeah.. we could have XXXConnection and XXX for a bit with XXXConnection deprecated